functional-samples/tutorial.focus-mode-debugging/README.md

```diff
@@ -0,0 +1,14 @@
+# Oliver Focus Mode
+
+Extension used in the [Debugging Chrome extensions](https://www.youtube.com/watch?v=Ta-YTDhiBIQ) DevTools Tips video. This extension simplifies the styling of the Extensions and Chrome Web Store documentation pages so that they are easier to read when the action icon is clicked.
```

Comment:
```suggestion
Extension used in the [Debugging Chrome extensions](https://www.youtube.com/watch?v=Ta-YTDhiBIQ) DevTools Tips video. This extension simplifies the styling of the extensions and Chrome Web Store documentation pages so that they are easier to read when the action icon is clicked.
```